components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LocalEntryResource.java

```diff
@@ -94,12 +94,11 @@ public boolean invoke(MessageContext messageContext,
         return false;
     }
 
-    private static List<Entry> getSearchResults(MessageContext messageContext, String searchKey) throws AxisFault {
+    private List <Entry> getSearchResults(MessageContext messageContext, String searchKey) throws AxisFault {
```

Comment:
Shall we remove this white space from here too?

Reply:
fixed